packages/content-conformance/src/cli.ts

```diff
@@ -19,32 +19,27 @@ yargs(hideBin(process.argv)).command(
       description: 'Path to config file',
       type: 'string',
     },
-    files: {
-      description: 'Path to content files',
-      default: [],
-      type: 'array',
-    },
   },
   async (argv): Promise<void> => {
+    const [...files] = <string[]>argv._
```

Comment:
Minor: if you are meaning to cast `argv._` here, I think this is more clear:
```suggestion
    const [...files] = argv._ as string[]
```